test/component/metrics/metrics_actions_test.js

```diff
@@ -231,8 +232,8 @@ describe('Metrics', function() {
                                 should.equal(m.services.unknownt.sum.actionEntityUpdate, 1);
                                 should.equal(m.services.unknownt.sum.okActionEntityUpdate, 0);
                                 should.equal(m.services.unknownt.sum.failedActionEntityUpdate, 1);
-                                should.equal(m.services.unknownt.sum.outgoingTransactions, 1);
-                                should.equal(m.services.unknownt.sum.outgoingTransactionsErrors, 1);
+                                //should.equal(m.services.unknownt.sum.outgoingTransactions, 1);
+                                //should.equal(m.services.unknownt.sum.outgoingTransactionsErrors, 1);
```

Comment:
Lefover comments? Remove/recover?

Reply:
Fixed in https://github.com/telefonicaid/perseo-fe/pull/717/commits/205e7238267077fe66931465fa0ee4f1e0e5d23c